src/ripple/protocol/Feature.h

```diff
@@ -34,6 +34,52 @@ uint256
 feature (const char* name);
 /** @} */
 
+constexpr size_t FeatureBitsetSize = 32;
```

Comment:
See the comment here for an idea for how to address this: https://github.com/ripple/rippled/pull/2108#discussion_r117269810